app/Providers/AppServiceProvider.php

```diff
@@ -31,5 +35,13 @@ public function boot(): void
             }
             Vite::useHotFile($hot2FilePath);
         }
+
+        Route::bind('id', function ($value) {
+            try {
+                return Uuid::fromString($value);
```

Comment:
ja bym te Uuidy wielkie olał, a tego bind'a zrobił żeby tylko walidował
return Uuid::fromString($value)->toString();

Reply:
i w ogóle czemu klasa tego typu nie jest stringable, jak już obiektowo, to obiektem który ma __toString :D